subsys/partition_manager/Kconfig

```diff
@@ -111,6 +111,10 @@ config PM_SINGLE_IMAGE
 
 DT_CHOSEN_EXT_FLASH:= nordic,pm-ext-flash
 
+config PM_EXTERNAL_FLASH_HAS_DRIVER
+	bool
```

Comment:
We could also make it non-anonymous and get rid off `PM_OVERRIDE_EXTERNAL_DRIVER_CHECK`, right?

Reply:
I would leave this hidden for now. Users can always redefine it in Kconfig for their project, the 'HAS' macros are usually hidden and once we make it visible.